### DISCUSSION TITLE
https://notes.alexkehayias.com/notion-rollup-count-by-month/

### DISCUSSION THREAD

**Discussion by giscus** [alexkehayias/note-comments#2] [Announcements]
# https://notes.alexkehayias.com/notion-rollup-count-by-month/

Notion has limited support for counting rows in a table using rollups.

https://notes.alexkehayias.com/notion-rollup-count-by-month/

**Comment by GeorgiEstonia** [comment]
Saved my life!!! I was trying to find a similar solution for weeks. Using it to calculate attendance and workers' hours by month.

**Reply by alexkehayias** [reply]
Glad to hear it!

**Comment by anishTPM** [comment]
In the last result, is it possible to have different view of months and for each month we can see the Month wise Count of Sum of individual Task Type . 

like, Jan will be one view, where only **Total** of January month will be visible for all 3 task type. The same way there will 11 more views will be there.

**Reply by alexkehayias** [reply]
I'm not sure I follow, but I think you could use the same technique to group by multiple criteria. I haven't done this before so I'm not sure, sorry!

**Reply by anishTPM** [reply]
Looks like this is not possible!!

**Reply by anishTPM** [reply]
Looks like Notion doesn't support this !
